EIPS/eip-4750.md

```diff
@@ -1,100 +1,89 @@
 ---
 eip: 4750
-title: EOF - Functions
-description: Individual sections for functions with `CALLF` and `RETF` instructions
+title: EOF - Modules and Procedures
+description: Individual sections for modules with `CALLPROC` and `RETURNPROC` instructions
```

Comment:
Not sure if markdown is supported in the preamble - you might want to change this.

Reply:
I just checked the original `CALLF` and `RETF` at https://eips.ethereum.org/EIPS/eip-4750.  They're not.

Reply:
```suggestion
description: Individual sections for modules with CALLPROC and RETURNPROC instructions
```